.env.example

```
@@ -13,6 +13,9 @@ PUBLIC_PROMPT_HASH_CONTRACT_ID=CXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
 PUBLIC_STELLAR_NATIVE_ASSET_CONTRACT_ID=CDLZFC3SYJYDZT7K67VZ75HPJVIEUVNIXF47ZG2FB2RMQQVU2HHGCYSC
 PUBLIC_STELLAR_SIMULATION_ACCOUNT=GXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
 PUBLIC_UNLOCK_PUBLIC_KEY=BASE64_PUBLIC_KEY
+PUBLIC_PINATA_JWT=replace-with-public-pinata-jwt-or-upload-token
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The PUBLIC_PINATA_JWT key should go before the PUBLIC_PROMPT_HASH_CONTRACT_ID key

(UnorderedKey)